ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -130,7 +129,7 @@ class ColorDemoController: UIViewController {
         super.viewDidAppear(animated)
         navigationController?.navigationBar.shadowImage = UIImage()
         if let window = view.window {
-            segmentedControl.selectedSegmentIndex = colorProviderThemedWindowTypes.firstIndex(where: { return window.isKind(of: $0.windowType) }) ?? 0
+            segmentedControl.selectedSegmentIndex = colorProviderThemes.firstIndex(where: { return window.isKind(of: $0.demoColorTheme.windowType) }) ?? 0
```

Comment:
I still see the segmented control jumps between default to green when loading the color demo controller. what is missing here to get the selected index correct?

Reply:
Two reasons:
1. It's being set after the view already appeared (in viewDidAppear)
2. After moving it to viewWillAppear or viewDidLoad, the green segment selector stayed on Default and didn't switch over (the isKind(of:) call returns the wrong index).

To fix this, I created a static variable in ColorDemoController that is set to the window type and check for that instead in the firstIndex(where:) call.